database/seed.js

```diff
@@ -18,35 +18,31 @@ connection.once('open', () => {
 
 var documents = [];
 
-var generateRandomDocs = function () {
-  for (var i=0; i<100; i++) {
-    var document = {};
+var potentialLocations = ['Santorini', 'Mykonos', 'Paros', 'Crete', 'Naxos', 'Corfu', 'Zakynthos', 'Milos', 'Hydra', 'Skiathos', 'Lefkada', 'Patmos', 'Delos', 'Kos', 'Icaria', 'Amorgos', 'Syros', 'Samos', 'Chios'];
 
-    var getRandomInt = function (max) {
-      return Math.floor(Math.random() * Math.floor(max));
-    };
+var potentialHousingTypes = ['Villa', 'Palace', 'Bungalow', 'Mansion', 'Escape', 'House', 'Vista'];
 
-    var potentialLocations = ['Santorini', 'Mykonos', 'Paros', 'Crete', 'Naxos', 'Corfu', 'Zakynthos', 'Milos', 'Hydra', 'Skiathos', 'Lefkada', 'Patmos', 'Delos', 'Kos', 'Icaria', 'Amorgos', 'Syros', 'Samos', 'Chios'];
+var potentialPrefixes = ['The', 'Wonderful', 'Magnificent', 'Quiet', 'Gorgeous'];
 
-    var potentialHousingTypes = ['Villa', 'Palace', 'Bungalow', 'Mansion', 'Escape', 'House', 'Vista'];
+var getRandomInt = function (max) {
+  return Math.floor(Math.random() * Math.floor(max));
+};
 
-    var potentialPrefixes = ['The', 'Wonderful', 'Magnificent', 'Quiet', 'Gorgeous'];
+var randomIntFromInterval = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
 
-    document.name = `${potentialPrefixes[getRandomInt(potentialPrefixes.length)]} ${potentialLocations[getRandomInt(potentialLocations.length)]} ${potentialHousingTypes[getRandomInt(potentialHousingTypes.length)]}`;
+for (var i=0; i<100; i++) {
+  var document = {};
 
-    var randomIntFromInterval = function (min, max) {
-      return Math.floor(Math.random() * (max - min + 1) + min);
-    };
+  document.name = `${potentialPrefixes[getRandomInt(potentialPrefixes.length)]} ${potentialLocations[getRandomInt(potentialLocations.length)]} ${potentialHousingTypes[getRandomInt(potentialHousingTypes.length)]}`;
```

Comment:
There is too much repetition here. Could you provide each value with a function that takes in potentialX as a parameter and returns a string?